src/probnum/diffeq/odefilter/init_routines/_autodiff.py

```diff
@@ -0,0 +1,247 @@
+"""Automatic-differentiation-based initialization routines."""
+
+import numpy as np
+
+from probnum import problems, randprocs, randvars
+
+from ._interface import InitializationRoutine
+
+
+class _AutoDiffBase(InitializationRoutine):
+    def __init__(self):
+        super().__init__(is_exact=True, requires_jax=True)
+        self._jax, self._jnp = self._import_jax()
+
+    def _import_jax(self):
+
+        # pylint: disable="import-outside-toplevel"
+        try:
+            import jax
+            import jax.numpy as jnp
+            from jax.config import config
+
+            config.update("jax_enable_x64", True)
+
+            return jax, jnp
+
+        except ImportError as err:
+            raise ImportError(self._import_jax_error_msg) from err
+
+    @property
+    def _import_jax_error_msg(self):
+        return (
+            "Cannot perform Jax-based initialization without the optional "
+            "dependencies jax and jaxlib. "
+            "Try installing them via `pip install jax jaxlib`."
+        )
```

Comment:
What is the argument for making this a property? I find that odd (from an OOP perspective)

Reply:
it is just a string, making this a function feels odd. I thought about making this a class variable, but this felt weird, too. A property that is stored right next to the import function felt like the cleanest solution

Reply:
Why does it have to belong to the object? Why not just define the string directly at the point where it is raised?
And if we need it multiple times and you don't want to paste it everywhere, we could define it as a global somewhere? I don't know I find that everything else feels kinda weird.